docs/DeveloperGuide.adoc

```diff
@@ -295,35 +311,99 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
 [width="59%",cols="22%,<23%,<25%,<30%",options="header",]
 |=======================================================================
 |Priority |As a ... |I want to ... |So that I can...
-|`* * *` |new user |see usage instructions |refer to instructions when I forget how to use the App
+|`* * *` |full-time university student without any stable income |record my expenses |track my spending and practice financial management skills
```

Comment:
User is not specific: The need to record expenses is not exclusive to a "full-time university student without any stable income"